src/MsiFileTests/WxsValidationTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Xml;
+
+namespace MsiFileTests
+{
+    [TestClass]
+    public class WxsValidationTests
+    {
+        [TestMethod]
+        public void AllDropFilesAreAccountedFor()
+        {
+            // Fails if any non-pdb drop files exist without a corresponding entry in either
+            // the WXS file or the set of intentional exclusions
+            string currentFolder = Path.GetFullPath(Assembly.GetExecutingAssembly().Location);
+            string repoRoot = Path.GetFullPath(Path.Combine(currentFolder, @"..\..\..\..\.."));
+            string wxsFile = Path.GetFullPath(Path.Combine(repoRoot, @"MSI\Product.wxs"));
+            HashSet<string> productComponentExclusions = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+            {
+                "AccessibilityInsights.Extensions.DiskLoggingTelemetry.dll",
+                "Axe.Windows.Automation.dll",
+            };
+
+            CompareWxsSectionToDropPath(repoRoot, @"AccessibilityInsights\bin\release\net472", wxsFile, "ProductComponent", productComponentExclusions);
+            CompareWxsSectionToDropPath(repoRoot, @"AccessibilityInsights\bin\release\net472\IssueTemplates", wxsFile, "IssueTemplates");
+            CompareWxsSectionToDropPath(repoRoot, @"AccessibilityInsights.VersionSwitcher\bin\release\net472", wxsFile, "VersionSwitcher");
```

Comment:
One potential drawback to this approach is that if we hypothetically add a new directory to the drop, we'll have to remember to add it here or else its files will go unvalidated. I could imagine some kind of logic where we ensure that every `Directory` node in the wxs file has been compared with a directory (such as a comparison between the total count of `Directory` nodes under `INSTALLFOLDER` vs the number of `GetFilesIncludedInWxsComponent` calls), but that might be excessive given that we almost never add new directories